recipes/spm1d/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "spm1d" %}
+{% set version = "0.4.0" %}
+{% set sha256 = "834195f92f19a68970194e61ec5862fe09ac14a4156cd8eef7e159805b33a3c0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - scipy
+    - numpy
+    - matplotlib
+    
+  run:
+    - python
+    - setuptools
+    - scipy
+    - numpy
+    - matplotlib
+
+test:
+  imports:
+    - spm1d
+
+about:
+  home: http://www.spm1d.org/
+  license: GPL
```

Comment:
Also can we please include a license file from the source directory using `license_file`.